spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -35,10 +35,9 @@
 		<arquillian.version>1.6.0.Final</arquillian.version>
 		<arquillian-cube.version>1.15.2</arquillian-cube.version>
 		<hoverfly.version>0.13.0</hoverfly.version>
-		<kubernetes-client.version>5.5.0</kubernetes-client.version>
-		<kubernetes-java-client.version>13.0.0</kubernetes-java-client.version>
+		<kubernetes-fabric8-client.version>5.9.0</kubernetes-fabric8-client.version>
+		<kubernetes-native-client.version>13.0.0</kubernetes-native-client.version>
 		<istio-client.version>1.7.7.1</istio-client.version>
-		<mockwebserver.version>0.1.2</mockwebserver.version>
```

Comment:
this dependency was never supposed to be here and I will explain why below.

The very first dependency we have in `spring-cloud-kubernetes-dependencies` is: 

```
<dependencyManagement>
		<dependencies>
			<dependency>
				<groupId>io.fabric8</groupId>
				<artifactId>kubernetes-client-bom</artifactId>
				<version>${kubernetes-fabric8-client.version}</version>
				<type>pom</type>
				<scope>import</scope>
			</dependency>
```

which is great, it means we rely on the on the bom that fabric8 provides. 

This bom has `io:fabric8:kubernetes-server-mock`, which internally provides: `io.fabric8:mockwebserver.version`, which means we should have never track this individually. I do not know the reasons it was added, but it surely does not matter at this point in time. We should not track that.
I also did a : 

```
bash-3.2$ read  -d ' ' x < <(grep -r "<artifactId>mockwebserver</artifactId>" .);echo "$x"
./spring-cloud-kubernetes-dependencies/pom.xml:                         <artifactId>mockwebserver</artifactId>
./spring-cloud-kubernetes-fabric8-config/pom.xml:                       <artifactId>mockwebserver</artifactId> 

``` 

To see where we have it as a dependency also. Since it was present in `spring-cloud-kubernetes-fabric8-config` (but not needed), I removed it from there also.